components/TodoTask.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import { TouchableOpacity, Text, View, Image } from 'react-native'
+
+export const TodoTask = ({ item, pressHandler }) => {
+
+    return (
+        <TaskContainer>
+            <TextContainer>
+                <TodoText>{item.task}</TodoText>
+            </TextContainer>
+            <TouchableText onPress={() => pressHandler(item.key)}>
+                <Checkmark>&#10004;</Checkmark>
+            </TouchableText>
+        </TaskContainer>
+    )
+} 
+
+const TaskContainer = styled.View`
+  flex-direction: row
+  margin-top: 20px;
+`
+const TextContainer = styled.View`
+  padding: 10px;
+  background-color: #e6e6e6
+  border-width: 1
```

Comment:
There is nothing like `border-width: 1`, I think you meant `border-width: 1px` - we need to remember about units